src/Geometry.ts

```diff
@@ -331,32 +331,54 @@ export enum IntersectionState {
 }
 
 export class Frustum {
-    // Left, Right, Top, Bottom, Near, Far
-    public planes: Plane[] = nArray(6, () => new Plane());
+    public planes: Plane[] = [];
+    public near: Plane;
+    public far: Plane;
+
+    constructor(nPlanes: number) {
```

Comment:
Mind defaulting this?

```suggestion
    constructor(nPlanes = 6) {
```